labs-ios-starter/Surveys/Model Controllers/TopicController.swift

```diff
@@ -122,7 +122,6 @@ class TopicController {
                     completion(.success(Void()))
                     return
                 }
-
                 self.deleteTopicsFromCoreData(topics: topicsNotOnServer, context: context)
```

Comment:
I like some white space in methods, especially after a block. Either way is fine, I just think some white space makes it more easily readable... and I think I removed this line?? lol!